content/en/real_user_monitoring/guide/devtools-tips.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Tips when using browser's developer tools
+kind: guide
+further_reading:
+    - link: '/real_user_monitoring/browser'
+      tag: 'Documentation'
+      text: 'RUM Browser Monitoring'
+    - link: '/logs/log_collection/javascript'
+      tag: 'Documentation'
+      text: 'Browser Log Collection'
+---
+
+## Overview
+
+This guide provides some tips when using the browser's developer tools to debug an application instrumented with the Datadog browser SDKs.
+
+## File and line number mismatch in DevTools console
+
+The browser SDK instruments the console functions (`console.error`, but also `.log`, `.info` and `.warn`) to collect data about the behavior of the application.
+This leads to inconvenient side-effects, as shown below:
+{{< img src="real_user_monitoring/guide/devtools-tips/issue_console.jpg" alt="Devtools console showing issue about wrong file and line numbers for a console.error statement.">}}
+
+In the picture above, the `console.error` function is instrumented. As a developer, it is expected to see the real file and line number on which this statement was called. However, instead of displaying `VM505:1`, the console shows `datadog-rum-v4.js:1`.
+
+### Add scripts to your browser ignore list
+
+Most browsers allow developers to select scripts and add them to an ignore list. Doing so for `datadog-rum-v*.js` and `datadog-logs-v*.js` ensures the DevTools display the proper file and line number.
+
+How to use this feature on Google Chrome:
+{{< img src="real_user_monitoring/guide/devtools-tips/script_ignore_list.jpg" alt="How to add script to the ignore list in Google Chrome.">}}
+
+In the console tab, expand the output from the console statement. Right-click on each script you wish to ignore, and select the option 'add script to ignore list'.
+Note: The ignore list can be managed in Developer Tools > Settings > Ignore List
+
+This method works well when using the CDN (sync/async) installation methods. If you are using the NPM package, make sure that you have sourcemaps enabled. Else, SDK code may be bundled with your application code, preventing DevTools to ignore the SDK.
```

Comment:
```suggestion
This method works well when using the [CDN (sync/async) installation methods][3]. If you are using the NPM package method, make sure that you have `sourcemaps` enabled. Otherwise, the SDK code may be bundled with your application code, preventing DevTools from ignoring the SDK.
```

Reply:
👌